src/crypto/index.js

```diff
@@ -1221,6 +1221,20 @@ Crypto.prototype.checkOwnCrossSigningTrust = async function() {
     if (oldSelfSigningId !== newCrossSigning.getId("self_signing")) {
         logger.info("Got new self-signing key", newCrossSigning.getId("self_signing"));
 
+        // Try to cache the self-signing private key as a side-effect
```

Comment:
I'm wondering why we need to cache the SSK & USK PKs as part of this change? Or is this unrelated to MSK key requests?

Reply:
I added after noticing that if you set up a new session via security phase, we were only retrieving the MSK and SSK. The USK was not retrieved since it's not actually used during login. By eagerly caching it here, we avoid another prompt when you go to verify a user, and also ensures we have the key stored to share with another device during verification.

Reply:
The SSK portion here is actually unneeded, as it will be implicitly retrieved as part of `signDevice` just below, but it felt clearer to explicitly cache the key, so we now have one flow where all 3 keys are being cached.